### ISSUE TITLE
Indicate range of values

### ISSUE THREAD

**Issue by bast** [densities/blob#15]
To make it easier to select an isovalue. Also document how the default isovalue is computed.

**Comment by stigrj** [comment]
It would be nice to have a scroll bar or something for the isovalue. At least something click-based.

**Comment by bast** [comment]
I agree. The meshing can still be a bit sluggish for a slider rule: for 80x80x80 points the meshing takes 1-2 seconds. But this is without any optimization and screening (#3). Once this is sped up a bit, the meshing could be done on the fly.
